piksi_tools/console/utils.py

```diff
@@ -32,71 +32,132 @@
 CODE_GPS_L5I = 9
 CODE_GPS_L5Q = 10
 CODE_GPS_L5X = 11
+CODE_GPS_L1CI = 56
+CODE_GPS_L1CQ = 57
+CODE_GPS_L1CX = 58
+CODE_AUX_GPS = 59
 
 CODE_GLO_L1OF = 3
 CODE_GLO_L2OF = 4
+CODE_GLO_L1P = 29
+CODE_GLO_L2P = 30
 
 CODE_SBAS_L1CA = 2
+CODE_SBAS_L5I = 41
+CODE_SBAS_L5Q = 42
+CODE_SBAS_L5X = 43
+CODE_AUX_SBAS = 60
 
-CODE_BDS2_B11 = 12
+CODE_BDS2_B1 = 12
 CODE_BDS2_B2 = 13
+CODE_BDS3_B1CI = 44
+CODE_BDS3_B1CQ = 45
+CODE_BDS3_B1CX = 46
+CODE_BDS3_B5I = 47
+CODE_BDS3_B5Q = 48
+CODE_BDS3_B5X = 49
+CODE_BDS3_B7I = 50
+CODE_BDS3_B7Q = 51
+CODE_BDS3_B7X = 52
+CODE_BDS3_B3I = 53
+CODE_BDS3_B3Q = 54
+CODE_BDS3_B3X = 55
```

Comment:
we will have to keep an eye out for a future RINEX spec in case they manage to pick different codes...

Reply:
oh yes, these are hit-and-miss right now...